coderdojochi/migrations/0005_auto_20160726_2010.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-07-26 20:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
@bkellgren Can you add code to convert the old M2M code data to the new pattern. We need to keep our old waitlist data.